setup.py

```diff
@@ -31,23 +31,22 @@
     packages=find_packages(where="src"),
     package_data={'wsproto': ['py.typed']},
     package_dir={'': 'src'},
-    python_requires='>=3.6.1',
+    python_requires='>=3.7.0',
     license='MIT License',
     classifiers=[
         'Development Status :: 5 - Production/Stable',
         'Intended Audience :: Developers',
         'License :: OSI Approved :: MIT License',
         'Programming Language :: Python',
         'Programming Language :: Python :: 3',
-        'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: 3.9',
+        'Programming Language :: Python :: 3.10',
         'Programming Language :: Python :: Implementation :: CPython',
         'Programming Language :: Python :: Implementation :: PyPy',
     ],
     install_requires=[
-        "dataclasses ; python_version < '3.7'",
```

Comment:
Is there a specific incompatibility with dataclasses on 3.6 that makes it easier to simply drop 3.6 and move to "native" 3.7?

Reply:
dataclasses was added to the stdlib in 3.7

Reply:
yes, my question was actually: why drop support for 3.6?
or was this needed to fix the compatibility with h11?

Reply:
oh right. 3.6 is now unsupported, https://endoflife.date/python. h11 will drop 3.6 in the next release as well.